shippable.yml

```diff
@@ -0,0 +1,76 @@
+language: c
+
+compiler:
+  - gcc
+
+runtime:
+  nodePool: shippable_shared_aarch64
+
+env:
+  global:
+    - secure: OuhkRMNLWCzMcJdPrUtvyo78/NqOtP1WHEfVViD5oCHIFr7rTjOTRFC+1Tcal//Z9XoqlGTBSfthqn6Z8QemqdljeVSXJgb9zXJMrq4Hl/hWrNhJ/M1m/rh30EQYnAzFyiTGN9yuzOLArLDq1Emv2dzDRpZybrx6FpSvo+hm3GQZNW55+KUu5uUoCsKZwSIjytgJPuec/sTOkV81OP2bszwqmvh+Xhb1UT4ukYlDKK3B+uggKc7MCnOWiYBkh5fsNmEjUiBS3tj4B2nD4rTv0isDucZJTe5aHbOFhjRi5Yh4yk4qxi4WqwOHYDTIU4rzEOuOKihVeo52IayuJE+4Iw==
+  matrix:
+    - LLVM_ENABLE_ASSERTIONS=ON
+    - LLVM_ENABLE_ASSERTIONS=OFF
+
+build:
+  ci:
+    # Install dependencies.
+    - export DEBIAN_FRONTEND=noninteractive
+    - apt-get -y update
+    - apt-get -yq install binutils-dev cmake curl git-core ninja-build p7zip-full
+    - update-alternatives --install /usr/bin/ld ld /usr/bin/ld.gold 99
+    # Build LLVM incl. LLD and compiler-rt.
+    - git submodule update --init --recursive
```

Comment:
No need, they already initialize all submodules in their git sync. Looks like they'll be adding shallow clones soon too, Shippable/support#3862, so we won't have to waste time and bandwidth downloading the entire repo for each CI run.

Reply:
I know, but I prefer it this way, in case someone (distro maintainer...) uses these files as basis for his own scripts. There's no need to set the `DEBIAN_FRONTEND` env variable either etc.

Reply:
Oh, OK, got it.